child_compassion/models/project_reservation.py

```diff
@@ -88,6 +88,21 @@ def cancel_reservation(self):
             'target': 'current'
         }
 
+    @api.multi
+    def show_reserved_children(self):
+
+        self.ensure_one()
+        return {
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'res_model': 'project.reservation.wizard',
```

Comment:
You don't need to create a new wizard for that. You can search the children directly inside this method and change this return with these values :
```
'res_model' : 'compassion.child',
'domain': [('id', 'in', found_children.ids)],
'view_mode': ['tree', 'form'],
```